tests/test_pollination.py

```diff
@@ -198,7 +198,7 @@ def test_pollination_no_farm_regression(self):
             result_sum += numpy.sum(data_block)
         # the number below is just what the sum rounded to two decimal places
         # when I manually inspected a run that appeared to be correct.
-        self.assertAlmostEqual(result_sum, 58.669518, places=2)
+        self.assertAlmostEqual(result_sum, 58.407155, places=2)

```

Comment:
Are these changing because of the change in how we're building kernels?  Or is it more about masking?  Either way, it might be useful to note in the HISTORY why values are changing slightly.

Reply:
This is because of the switch to `pygeoprocessing.kernels`, which is more correct in some cases. The kernel implementation in pollination didn't always produce symmetrical kernels:
<img width="377" alt="image" src="https://github.com/natcap/invest/assets/43770515/703f3274-b92e-4ab0-96f2-0b216b3ac1fc">
With `pygeoprocessing.kernels`:
<img width="347" alt="image" src="https://github.com/natcap/invest/assets/43770515/c372ab05-d45e-4a77-bf8d-28f4c7efbf8d">

Reply:
Added a note to history.